source/extensions/filters/network/thrift_proxy/router/router_impl.h

```diff
@@ -164,7 +164,13 @@ class RouteMatcher {
   COUNTER(route_missing)                                                                           \
   COUNTER(unknown_cluster)                                                                         \
   COUNTER(upstream_rq_maintenance_mode)                                                            \
-  COUNTER(no_healthy_upstream)
+  COUNTER(no_healthy_upstream)                                                                     \
+  COUNTER(request_call)                                                                            \
+  COUNTER(request_oneway)                                                                          \
+  COUNTER(request_invalid_type)                                                                    \
+  COUNTER(response_reply)                                                                          \
+  COUNTER(response_exception)                                                                      \
+  COUNTER(response_invalid_type)
```

Comment:
while we are here, do we also want to bytes per request/response?

Reply:
I can do that in a follow-up PR, I just want to cover messageTypes for now (as we don't have any per-upstream metrics to begin with).